csharp/test/Ice/adapterDeactivation/AllTests.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) ZeroC, Inc.
 
+using Ice.Internal;
+
 namespace Ice
 {
     namespace adapterDeactivation
```

Comment:
Despite the name, these are really object adapter tests - not deactivation-specific.